standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/TableFields.java

```diff
@@ -58,9 +60,9 @@
 
     private static final ImmutableSet<String> allMultiValuedFields = new ImmutableSet.Builder<String>()
             .add("values")
-            .add("sd.cols.name")
-            .add("sd.cols.type")
-            .add("sd.cols.comment")
+            .add("sd.cd.cols.name")
```

Comment:
@vnhive StorageDescriptor seems to have a variable "cols" but not the actual ColumnDescriptor "cd". How does this work?

Reply:
Sorry about the confusion naveen. I have reverted these back to the StorageDescriptor attributes. Thank you for the comment.